File: 026_Remove_Duplicates_from_Sorted_Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        
        numTable = {}
        count = 0
        m = len(nums)
        prev = None
            
        for n in range(0,m):
            if nums[0] is None or nums[0] > prev:
                nums.append(nums[0])
                prev = nums[0]
                count += 1
            nums.pop(0)
                
        return count

File: 026_Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        
        int prev = NULL, count = 0, len = nums.size();
        
        for(int i=0; i<len; i++){
            if(i==0 || prev!=nums[0]){
                count++;
                nums.push_back(nums[0]);
                prev = nums[0];
            }
            nums.erase(nums.begin());
        }
        return count;
        
    }
};
